learn/on-premise/observability/metrics-collection/alerts.md

```diff
@@ -85,3 +85,17 @@ Following are the list of alerts included in WaveMaker
 :::note
 The Alerts listed above keep changing as new things are discovered and might vary over time
 :::
+
+### Configure Notifications for Alerts
+Prometheus uses alert manager to send notifications.In WaveMaker Enterprise Setup  alerts are configured through Email channel. Below link explains configuration of alerts using various channels like Slack ,Pageduty etc.                                                             
```

Comment:
Prometheus uses alert manager to send notifications. In WaveMaker Enterprise Setup  alert notifications are configured through Email channel by default. Below link explains configuration of alert notifications using various other channels like Slack ,Pageduty etc.